src/components/Header.jsx

```diff
@@ -1 +1,13 @@
-// implement Header component here
+import React from 'react';
+
+class Header extends React.Component {
```

Comment:
Lima, será que ficaria melhor, seguirmos o padrão de exportar o componente direto na classe ? 

```suggestion
export class Header extends React.Component {
```